packages/mural-canvas/src/lib/rpc.ts

```diff
@@ -181,6 +193,19 @@ export default class RpcClient extends EventEmitter {
     this.dispatch();
   };
 
+  rpc_callback(message: RpcMessage, error?: string, context?: RpcContext) {
```

Comment:
**nitpick**: I would like the interface to feel happier here, as sending `error: null`, when you want to send `context: {}` seems like a thorn in the side.

What do you think of using an object here as keyword params?
```
rpc_callback(message: RpcMessage, reply?: { error?: string, context?: RpcContext });

# invocations looks quite a bit better
const msg: RpcMessage = …;

rpc_callback(msg); // ACK
rpc_callback(msg, { context: { result: 42 } }); // ACK w/ context
rpc_callback(msg, { error: "I'm not a teapot!" }); // ACK w/ error
```

---

Another option would be to have `rpc_callback_error` to deal with the error case independently.

```
rpc_callback(msg, context);
rpc_callback_error(msg, error, context);
```

Reply:
> an object [...] as keyword params

That's much nicer. I updated this.